test/libsolidity/SolidityNameAndTypeResolution.cpp

```diff
@@ -1395,6 +1395,39 @@ BOOST_AUTO_TEST_CASE(events_with_same_name)
 	BOOST_CHECK(success(text));
 }
```

Comment:
Can you add two more overloading examples:
```
event A(uint, uint);
event A(uint, bytes);
```
and
```
event A(uint);
event A(bytes);
```

Reply:
Sure!